make/ha/sample/ha_storage.cfg

```diff
@@ -0,0 +1,50 @@
+# Uncomment the storge provide for your HA environment.
```

Comment:
Do you plan to use a script to copy the chunk to config_ha.yml?

Or do you plan to just mention in the doc and let user manually update?

Reply:
When install with HA, the config_ha.yml will be placed to ~/common/config/registry/config.yml. It will be done by script.

This is because we have different config.yml for HA mode and standalone mode